### PR TITLE
Add Governance page

### PR DESCRIPTION
**Pull request by sedders123**
Adding a governance page as requested by the trustees. Content originally came from the trustees I believe.

### Screenshots
<details><summary>Desktop</summary>

screencapture-localhost-4000-website-governance-2022-08-04-21_40_51

</details>

<details><summary>Mobile
</summary>

screencapture-localhost-4000-website-governance-2022-08-04-21_42_41 (1)

</details>

### CODE REVIEW

File: _sass/governance.scss
Comment (PeterJCLaw) [inline _sass/governance.scss:2]: While using `&` like this to join within a word is clever, I'd discourage doing so as it's clearer not to. The duplication of `trustee` in `trustee-row` and `trustee-avatar` is pretty minimal and allows for better discoverability (by searching for the exact class name).

**Review by PeterJCLaw — COMMENTED** [review]
Generally good, see suggestion about SASS usage though.

File: _sass/governance.scss
Comment (sedders123) [reply _sass/governance.scss]: 👍 29ce178

**Review by PeterJCLaw — DISMISSED** [review]
:+1: not sure if you want to wait for approval from a trustee though?

**Comment by sedders123** [comment]
> 👍 not sure if you want to wait for approval from a trustee though?

Thanks Peter! Yeah will wait to get an ok from a Trustee before merging :)

**Comment by Scarzy** [comment]
Personally, I would drop the word "legal" from before the constitution, it feels unnecessary and makes the sentence more awkward. I'll leave that judgement to you

Otherwise, consider this approved

**Review by PeterJCLaw — APPROVED** [review]

**Pull request merged**
